src/app.js

```diff
@@ -1,21 +1,34 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+  const normalizeURL = new URL(req.url, `http://${req.headers.host}`);
+  const fileName = normalizeURL.pathname.slice(6) || 'index.html';
+
+  if (!normalizeURL.pathname.startsWith('/file/')) {
+    res.statusCode = 404;
+
+    res.end(
+      'To load files, use the /file/ prefix followed by the path to the file.'
+    );
+  }
+
+  fs.readFile(`./public/${fileName}`, (error, data) => {
+    if (!error) {
+      res.statusCode = 200;
+      res.end(data);
+    }
+
+    res.statusCode = 404;
```

Comment:
Add an else block after checking if there is no error in the readFile callback. This ensures that the 404 status code and response are only sent when an error occurs.